models/comments.model.js

```diff
@@ -21,4 +22,19 @@ exports.getCommentsForArticle = ( articleId ) => {
             return comments
         }
     })
+}
+
+exports.addCommentToArticleForUser = (comment, article_id, username) => {
+    if (!username || !comment) return Promise.reject({ status: 400, msg: 'Missing inputs' })
```

Comment:
If the body is invalid PSQL will throw and error which can be handled in the app by middleware error handlers, this would be a DRYer approach and save you writing custom errors unnecessarily